Swap Fn and CtlL keys so that the physical layout becomes CtlL, Fn (how it should be).

This is just a forward ported version of the patch originally created by
middleton for the x200.7XHT24WW.img. The custom subroutine had to be moved
because 0x1b670 is now used by the factory fw. Subsequently, the two jump to
subroutine instructions had to be updated to pointed to the new address.

Refer to ../x200.7XHT24WW.img.d/008_fn_ctrl_swap.patch for details.

--- x200.7XHT25WW.img.orig.hex	2017-06-24 18:15:37.216544046 +0200
+++ x200.7XHT25WW.img.hex	2017-06-24 18:15:37.179544046 +0200
@@ -1627,7 +1627,7 @@
 000065a0  00 ff d5 13 5e 00 66 cc  f0 14 00 00 00 00 1a 00  |....^.f.........|
 000065b0  46 f8 7e 16 73 50 47 08  79 00 00 01 5e 00 0c 10  |F.~.sPG.y...^...|
 000065c0  7e 16 73 70 47 04 5a 00  66 76 7e 16 73 60 47 7c  |~.spG.Z.fv~.s`G||
-000065d0  7e 16 77 50 7e 16 75 40  45 70 21 bb 17 01 6a 29  |~.wP~.u@Ep!...j)|
+000065d0  7e 16 77 50 7e 16 75 40  45 70 5e 01 b8 10 6a 29  |~.wP~.u@Ep^...j)|
 000065e0  00 ff d5 0f 1c 19 46 54  6a 29 00 ff d5 10 0a 09  |......FTj)......|
 000065f0  6a a9 00 ff d5 10 6a 20  00 ff d5 12 1c 09 45 3a  |j.....j ......E:|
 00006600  6a 28 00 ff d5 13 f0 00  79 08 00 00 78 00 6a 29  |j(......y...x.j)|
@@ -1640,7 +1640,7 @@
 00006670  77 50 7f 16 67 40 f0 0d  55 52 7e 16 77 50 7f 16  |wP..g@..UR~.wP..|
 00006680  67 d0 01 00 6d 72 01 00  6d 71 04 80 6a 18 fd 44  |g...mr..mq..j..D|
 00006690  70 00 01 00 6d 70 56 70  6a 28 00 ff d5 13 f0 00  |p...mpVpj(......|
-000066a0  79 08 00 00 78 00 6a 29  00 ff da 70 21 bb 17 01  |y...x.j)...p!...|
+000066a0  79 08 00 00 78 00 6a 29  00 ff da 70 5e 01 b8 10  |y...x.j)...p^...|
 000066b0  1c 91 47 14 7f 16 70 60  6a a1 00 ff d5 0f f0 00  |..G...p`j.......|
 000066c0  6a a0 00 ff d5 10 40 02  55 3a 54 70 7a 01 00 00  |j.....@.U:Tpz...|
 000066d0  66 e4 10 00 08 09 91 00  44 02 0b 59 69 10 30 b0  |f.......D..Yi.0.|
@@ -2794,7 +2794,7 @@
 0000ae90  7f 26 72 10 6a 18 fe 54  72 10 54 70 7f 90 72 50  |.&r.j..Tr.Tp..rP|
 0000aea0  7f 26 70 00 7f 91 72 50  56 70 6a 18 fe 55 72 10  |.&p...rPVpj..Ur.|
 0000aeb0  7f 26 70 10 56 70 7e 26  73 50 46 04 7e 26 73 30  |.&p.Vp~&sPF.~&s0|
-0000aec0  54 70 7e c1 77 a0 67 08  6a 10 fe 0c 77 90 67 18  |Tp~.w.g.j...w.g.|
+0000aec0  54 70 7e 16 77 00 67 08  6a 10 fe 0c 77 90 67 18  |Tp~.w.g.j...w.g.|
 0000aed0  6a 10 fe 4b 77 00 67 28  7e c1 77 90 67 38 7e c1  |j..Kw.g(~.w.g8~.|
 0000aee0  77 80 67 48 7e bd 77 a0  67 58 6a 10 fe 4e 77 b0  |w.gH~.w.gXj..Nw.|
 0000aef0  67 78 6a 10 fe 0c 77 80  67 00 6a 10 fe 47 77 90  |gxj...w.g.j..Gw.|
@@ -7039,9 +7039,9 @@
 0001b7e0  00 00 00 00 00 01 a3 b0  00 01 a6 14 00 00 00 00  |................|
 0001b7f0  00 00 00 00 00 00 00 00  00 00 e5 ae 00 00 4b 70  |..............Kp|
 0001b800  00 00 e5 b4 00 01 4c 2e  ff ff ff ff ff ff ff ff  |......L.........|
-0001b810  ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff  |................|
-0001b820  ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff  |................|
-0001b830  ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff  |................|
+0001b810  21 bb 17 01 01 00 6d f0  6a 28 00 ff d5 13 a8 0f  |!.....m.j(......|
+0001b820  46 0c 77 01 7f 16 67 00  7e c1 77 a0 67 01 01 00  |F.w...g.~.w.g...|
+0001b830  6d 70 54 70 ff ff ff ff  ff ff ff ff ff ff ff ff  |mpTp............|
 0001b840  ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff  |................|
 0001b850  ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff  |................|
 0001b860  ff ff ff ff ff ff ff ff  ff ff ff ff ff ff ff ff  |................|
